torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -9,16 +9,19 @@
 from itertools import chain
 from typing import Any, Callable, Dict, List, Optional, Type
 
+import logging
 import torch
 import torch.distributed as dist
 from torch.optim import Optimizer
-import logging
 
 __all__ = ["ZeroRedundancyOptimizer"]
 
 
 # Credits:  classy_vision/generic/distributed_util.py
-def _recursive_copy_to_device(value: Any, non_blocking: bool, device: torch.device) -> Any:
+def _recursive_copy_to_device(
+    value: Any, non_blocking: bool,
```

Comment:
shall we use one arg per line format to stay consistent with other APIs in this file and other files?